scipy/integrate/_tanhsinh.py

```diff
@@ -971,24 +987,30 @@ def _nsum(f, a, b, step=1, args=(), log=False, maxterms=int(2**20), atol=None,
     Returns
     -------
     res : _RichResult
-        An instance of `scipy._lib._util._RichResult` with the following
+        An object similar to an instance of `scipy.optimize.OptimizeResult` with the
         attributes. (The descriptions are written as though the values will be
         scalars; however, if `func` returns an array, the outputs will be
-
         arrays of the same shape.)
+
         success : bool
-            ``True`` when the algorithm terminated successfully (status ``0``).
-        status : int
+            ``True`` when the algorithm terminated successfully (status ``0``);
+            ``False`` otherwise.
+        status : int array
             An integer representing the exit status of the algorithm.
-            ``0`` : The algorithm converged to the specified tolerances.
-            ``-1`` : Element(s) of `a`, `b`, or `step` are invalid
-            ``-2`` : Numerical integration reached its iteration limit; the sum may be divergent.
-            ``-3`` : A non-finite value was encountered.
-        sum : float
+
+            - ``0`` : The algorithm converged to the specified tolerances.
+            - ``-1`` : Element(s) of `a`, `b`, or `step` are invalid
+            - ``-2`` : Numerical integration reached its iteration limit; 
+              the sum may be divergent.
+            - ``-3`` : A non-finite value was encountered.
+
+        sum : float array
             An estimate of the sum.
-        error : float
-            An estimate of the absolute error, assuming all terms are non-negative.
-        nfev : int
+        error : float array
+            An estimate of the absolute error, assuming all terms are non-negative,
+            the function is computed exactly, and direct summation is accurate to
+            the precision of the result dtype.
+        nfev : int array
             The number of points at which `func` was evaluated.
 
     See Also
```

Comment:
Fix CI failure by removing `tanhsinh`. Should we add intersphinx mapping for `mpmath` and link to [`mpmath.nsum`](https://mpmath.org/doc/current/calculus/sums_limits.html#nsum)?

Reply:
Yes 👍

Reply:
Do you know how to determine what URL should be entered in the mapping? I've guessed and checked before.

Reply:
Yep you can check with `python -m sphinx.ext.intersphinx https://mpmath.org/doc/current/objects.inv` and then if it returns then the URL is fine and remove the object part.

So it will be `https://mpmath.org/doc/current`

Reply:
OK, and you just guessed `https://mpmath.org/doc/current` as the prefix based on the URL of the [nsum documentation](https://mpmath.org/doc/current/calculus/sums_limits.html#nsum)?

Reply:
Running the command gives you the list of all the refs. So in this case it returns

```
...
mpmath.nsum    :calculus/sums_limits.html#mpmath.nsum
...
```

So I think you can reference it with just `mpmath.nsum`

Reply:
Ah sorry, I guessed https://mpmath.org/doc/current as it's the base URL of the doc and normally the objects.inv is at the root of the doc.